app/assets/javascripts/miq_application.js

```diff
@@ -1456,8 +1458,14 @@ function miqAccordionSwap(_collapse, expand) {
 
 // This function is called in miqOnLoad
 function miqInitToolbars() {
-  $("#toolbar button:not(.dropdown-toggle), #toolbar ul.dropdown-menu > li > a, #toolbar .toolbar-pf-view-selector > ul.list-inline > li > a").off('click');
-  $("#toolbar button:not(.dropdown-toggle), #toolbar ul.dropdown-menu > li > a, #toolbar .toolbar-pf-view-selector > ul.list-inline > li > a").click(miqToolbarOnClick);
+  $("#toolbar:not(.miq-toolbar-menu) button:not(.dropdown-toggle), "+
+  "#toolbar:not(.miq-toolbar-menu) ul.dropdown-menu > li > a, "+
+  "#toolbar:not(.miq-toolbar-menu) .toolbar-pf-view-selector > ul.list-inline > li > a"
+  ).off('click');
+  $("#toolbar:not(.miq-toolbar-menu) button:not(.dropdown-toggle), " +
+  "#toolbar:not(.miq-toolbar-menu) ul.dropdown-menu > li > a, "+
+  "#toolbar:not(.miq-toolbar-menu) .toolbar-pf-view-selector > ul.list-inline > li > a"
+  ).click(miqToolbarOnClick);
```

Comment:
Minor, but, the two sets of selectors look the same, so maybe we don't have to have them twice and can simply `$(...).off('click').on('click', miqToolbarOnClick);`  (can you change `click(..)` -> `on('click', ..)` please too?)

And I think we don't really have to split that line, but if you do, might make sense to align the opening quotes...